File: JavaScript_Courses/JavaScript_Basics/Preparation/01_First_Steps_In_Coding_Exercise/09_Fish_Tank.js
function fishTank(input) {
    let length = Number(input[0]);
    let width = Number(input[1]);
    let heigth = Number(input[2]);

    let percentOff = Number(input[3]);

    let totalVolume = length * width * heigth / 1000;
    let activeVolume = totalVolume * (1 - percentOff / 100);

    console.log(activeVolume);
}


// fishTank(["85", "75", "47", "17"]);
// fishTank(["105", "77", "89", "18.5"]);

File: Python_Courses/Python_Basics/Python_PB_2024_10/01_First_Steps_in_Coding_Exercise/09_Fish_Tank.py
# Един литър вода се равнява на един кубичен дециметър/ 1л=1 дм3/.

length = int(input())  # cm
width = int(input())
height = int(input())
not_empty_percent = float(input())

total_volume = length * width * height  # in cm * cm * cm == cm^3
total_volume_in_liters = total_volume / 1000  # 1l == 1 dm^3 == 1000 cm^3

total_liters_needed = total_volume_in_liters * (1 - not_empty_percent / 100)  # 17 % => 0.17

print(total_liters_needed)
